### PR TITLE
Fix build and tests on *nix.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @pranavkm

### CODE REVIEW

File: src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public static class CommonUtilities
             {
                 using (var pdb = new MemoryStream())
                 {
-                    if (PlatformHelper.IsMono)
+                    if (PlatformHelper.IsMono || !PlatformHelper.IsWindows)
```
Comment (pranavkm) [inline src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs]: We probably don't care about the pdbs for scaffolded items in general. I think it should be ok to remove the else entirely.

**Comment by pranavkm** [comment]
:shipit:

File: src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs (hunk 1 above)
Comment (cesarbs) [inline src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs]: Wouldn't it be better to run this by other folks first?

Comment (pranavkm) [inline src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs]: Sure. cc @prafullbhosale, thoughts on this?

Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs]: Yeah, we shouldn't care about the pdbs. These assemblies are generated either for retrieving EF related information or to compile the razor templates. 
Should be safe to always pass pdbstream as null

**Comment by cesarbs** [comment]
Ping. Please take another look now that I've removed the pdb stream stuff.

**Comment by pranavkm** [comment]
Nice. :shipit:

**Comment by prafullbhosale** [comment]
:shipit:

**Pull request merged**